pkg/gofr/external_db_test.go

```diff
@@ -40,6 +40,7 @@ func TestApp_AddMongo(t *testing.T) {
 
 		mock.EXPECT().UseLogger(app.Logger())
 		mock.EXPECT().UseMetrics(app.Metrics())
+		mock.EXPECT().UseTracer(gomock.Any())
```

Comment:
please correct me if I am wrong.. but I assume `app.Tracer()` would be better instead of `gomock.Any()` just like we are using Logger and Metrics.

Reply:
Currently `app` doesn't have a method called `Tracer`.  Although it has `initTracer` but it serves a different purpose